### ISSUE TITLE
Clarification on Inference Process for Videos with More than 8 Phonemes

### ISSUE THREAD

**Issue by Hu-bc** [MarshallT-99/VALLR#2]
Hi,
In your paper, the [inference](https://github.com/MarshallT-99/VALLR/blob/9793489136bc00293242a5410def05ba2806e16f/main.py#L521) process for the test set indicates that the phoneme labels often exceed 8 phonemes. However, according to the provided inference code, each video is transformed into a fixed 16-frame input, followed by temporal downsampling. This results in only 8 phoneme logits being produced during the CTC decoding phase. It appears that each video, regardless of the number of frames, can output at most 8 phonemes.

This raises a concern because many of the videos in the test set contain more than 8 phonemes. Given this limitation, I am curious how the inference process is intended to handle these videos.
Thanks,
Bochao
